desc/objectives/objective_funs.py

```diff
@@ -700,7 +701,7 @@ def things(self):
             RuntimeError,
             "ObjectiveFunction must be built with ObjectiveFunction.build() first",
         )
-        return self._flatten(self._all_things)
+        return self._flatten(self.all_things)
 
     @things.setter
     def things(self, new):
```

Comment:
I think it might be worth it to just get rid of this setter method. Changing around what things are bound to an equilibrium is likely to cause lots of weird problems, better to just make a new objective.